x-pack/plugins/cases/server/saved_object_types/telemetry.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsType } from 'src/core/server';
+import { CASE_TELEMETRY_SAVED_OBJECT } from '../../common/constants';
+
+export const casesTelemetrySavedObjectType: SavedObjectsType = {
+  name: CASE_TELEMETRY_SAVED_OBJECT,
+  hidden: false,
+  namespaceType: 'agnostic',
+  mappings: {
```

Comment:
Do we not need mappings for the saved object fields?

Reply:
No, because we do not want to increase the count of the attributes in the Kibana index. We will never do any aggregation to this saved object in Kibana. It is just a placeholder for data.